src/nvim/event/socket.c

```diff
@@ -45,6 +45,12 @@ int socket_watcher_init(Loop *loop, SocketWatcher *watcher,
       return UV_EINVAL;
     }
 
+    if (*port == NUL) {
```

Comment:
Should this also be used if `iport` is `0` or is `"0"` always an acceptable port description?